thor/request.py

```diff
@@ -6,15 +6,41 @@
 from datetime import timedelta
 import numpy as np
 import logging
+from math import floor
 
 
 def getList(dictTree,
```

Comment:
This is old but I do not like this function name. Maybe "getReaderList" or something like that is a better description?